packages/engine/Source/Core/ITwinPlatform.js

```diff
@@ -0,0 +1,313 @@
+ *
+ * @type {string|undefined}
+ */
+ITwinPlatform.defaultAccessToken = undefined;
+
+/**
+ * Gets or sets the default iTwin API endpoint.
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @type {string|Resource}
+ * @default https://api.bentley.com
+ */
+ITwinPlatform.apiEndpoint = new Resource({
+  url: "https://api.bentley.com",
+});
+
+/**
+ * Get the export object for the specified export id
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @param {string} exportId
+ *
+ * @throws {RuntimeError} Unauthorized, bad token, wrong scopes or headers bad.
+ * @throws {RuntimeError} Requested export is not available
+ * @throws {RuntimeError} Too many requests
+ * @throws {RuntimeError} Unknown request failure
+ */
+ITwinPlatform.getExport = async function (exportId) {
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.string("exportId", exportId);
+  if (!defined(ITwinPlatform.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+  //>>includeEnd('debug')
+
+  const headers = {
+    Authorization: `Bearer ${ITwinPlatform.defaultAccessToken}`,
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+  };
+
+  // obtain export for specified export id
+  const url = `${ITwinPlatform.apiEndpoint}mesh-export/${exportId}`;
+
+  // TODO: this request is _really_ slow, like 7 whole second alone for me
+  // Arun said this was kinda normal but to keep track of the `x-correlation-id` of any that take EXTRA long
+  const response = await fetch(url, { headers });
+  if (!response.ok) {
+    const result = await response.json();
+    if (response.status === 401) {
+      throw new RuntimeError(
+        `Unauthorized, bad token, wrong scopes or headers bad. ${result.error.details[0].code}`,
+      );
+    } else if (response.status === 404) {
+      throw new RuntimeError(`Requested export is not available ${exportId}`);
+    } else if (response.status === 429) {
+      throw new RuntimeError("Too many requests");
+    }
+    throw new RuntimeError(`Unknown request failure ${response.status}`);
+  }
+
+  /** @type {ExportResponse} */
+  const result = await response.json();
+  return result;
+};
+
+/**
+ * Get the list of exports for the given iModel + changeset
+ *
+ * @experimental This feature is not final and is subject to change without Cesium's standard deprecation policy.
+ *
+ * @param {string} iModelId
+ * @param {string} [changesetId]
+ *
+ * @throws {RuntimeError} Unauthorized, bad token, wrong scopes or headers bad.
+ * @throws {RuntimeError} Not allowed, forbidden
+ * @throws {RuntimeError} Unprocessable Entity
+ * @throws {RuntimeError} Too many requests
+ * @throws {RuntimeError} Unknown request failure
+ */
+ITwinPlatform.getExports = async function (iModelId, changesetId) {
```

Comment:
You should combine `loadExports` and `getExports`, by using:

![image](https://github.com/user-attachments/assets/faeacbee-49be-43a6-8f42-e1260cbfb458)

That will give you the url of the exports.
`get-export` will probably be deprecated too. It has performance problems because we will drop it soon, so I'd just use `get-exports`with `return=representation`